virtualization/windowscontainers/container-networking/network-drivers-topologies.md

```diff
@@ -13,14 +13,14 @@ ms.assetid: 538871ba-d02e-47d3-a3bf-25cda4a40965
 
 # Windows Container Network Drivers  
 
-In addition to leveraging the default 'nat' network created by Docker on Windows, users can define custom container networks. User-defined networks can be created using using the Docker CLI, [`docker network create -d <NETWORK DRIVER TYPE> <NAME>`](https://docs.docker.com/engine/reference/commandline/network_create/), command. On Windows, the following network driver types are available:
+In addition to leveraging the default 'nat' network created by Docker on Windows, users can define custom container networks. User-defined networks can be created using using the Docker CLI [`docker network create -d <NETWORK DRIVER TYPE> <NAME>`](https://docs.docker.com/engine/reference/commandline/network_create/) command. On Windows, the following network driver types are available:
 
 - **nat** – containers attached to a network created with the 'nat' driver will receive an IP address from the user-specified (``--subnet``) IP prefix. Port forwarding / mapping from the container host to container endpoints is supported.
 > Note: Multiple NAT networks are supported if you have the Windows 10 Creators Update installed!
 
 - **transparent** – containers attached to a network created with the 'transparent' driver will be directly connected to the physical network. IPs from the physical network can be assigned statically (requires user-specified ``--subnet`` option) or dynamically using an external DHCP server. 

```

Comment:
Please make this read, "... created with the 'transparent' driver will be directly connected to the physical network through an external Hyper-V switch"